modules/docs/arrow-docs/docs/_data/doc-versions.yml

```diff
@@ -1,22 +1,17 @@
 docVersions:
 
     - title: "v0.9.0"
-      url: https://0-9-0.arrow-kt.io/
-      previous: https://0-8-2.arrow-kt.io/
+      url: https://arrow-kt.io/0.9/
+      previous: https://arrow-kt.io/0.8/
 
     - title: "v0.10.4"
       url: https://arrow-kt.io/
-      previous: https://0-9-0.arrow-kt.io/
+      previous: https://arrow-kt.io/0.9/
 
     - title: "v0.10.5-SNAPSHOT"
-      url: https://next.arrow-kt.io/
+      url: https://arrow-kt.io/next/
```

Comment:
I only doubt regarding this change (and the same change on ` modules/docs/arrow-docs/docs/_data/commons.yml`). Do we have to change the sync with S3? We're synchronizing the documentation with `s3://$S3_BUCKET`. Should it be changed to `s3://$S3_BUCKET/next`? cc: @franciscodr

Reply:
Uhmmm, maybe that change is not needed?  The Jekyll build will generate a directory containing that version site, so the sync could be made at the root level?  Well, clearly @franciscodr has mor insight on this.